frontend/src/general/dashboard/DashboardCard.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./dashboard.css";
+import { Typography, Paper } from "@material-ui/core";
+
+export default function DashboardCard(props) {
+  const { title, children } = props;
+  return (
+    <>
+      <Paper className="dashboardCard" elevation={2}>
+        <Typography variant="h5" gutterBottom>
+          {title}
+        </Typography>
+        {children}
+      </Paper>
+    </>
+  );
+}
+
+DashboardCard.propTypes = {
+  title: PropTypes.string.isRequired,
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node
+  ])
```

Comment:
Instead of allowing 2 possible prop types we could pass an array of 1 node whenever we would otherwise pass a node. Nitpicking so feel free to ignore.

Reply:
I think you misunderstand. `children` refer to inner component as shown here: 
- singular node: ` <Parent><Child/></Parent>`
- array of nodes: ` <Parent><Child/><Child/><Child/></Parent>`
what you are suggesting does not make sense